logs/.placeholder

```diff
@@ -0,0 +1 @@
+# This is the default directory for logs. Placeholder so that directory is kept in git.
```

Comment:
Usually pretty common to use `.gitkeep` instead of `.placeholder` so we don't need to write a line in the file, but it's fine if you keep it like this too. Feel free to merge or change then merge

Reply:
Oh, I didn't know about this convention. Will rename files and stick to it.